spec.html

```diff
@@ -43695,6 +43745,10 @@ <h1>
         <emu-alg>
           1. Let _elementSize_ be the Element Size value specified in <emu-xref href="#table-the-typedarray-constructors"></emu-xref> for Element Type _type_.
           1. If _isLittleEndian_ is *false*, reverse the order of the elements of _rawBytes_.
+          1. If _type_ is ~float16~, then
+            1. Let _value_ be the byte elements of _rawBytes_ concatenated and interpreted as a little-endian bit string encoding of an IEEE 754-2019 binary16 value.
+            1. If _value_ is an IEEE 754-2019 binary16 NaN value, return the *NaN* Number value.
```

Comment:
```suggestion
            1. If _value_ is an IEEE 754-2019 binary16 NaN value, return *NaN*.
```

I don't see a reason to stray from our normal editorial conventions here.

Reply:
This currently matches the phrasing in the ~float32~ branch immediately below, which seems more important. We can later change all of them to match but I don't want to introduce an inconsistency within this method.